### DISCUSSION TITLE
Inconsistent freemod naming

### DISCUSSION THREAD

**Discussion by timiimit** [ppy/osu#23579] [Ideas] [answered]
There are different names given to freemods. On beatmap selection screen they are "freemods" as on stable, but in the room they are named "extra mods". I think they should have a consistent name to avoid confusion. Personally i would stick with "freemods".

song select:
![image](https://github.com/ppy/osu/assets/32331609/76cf0de5-f05d-4b9d-a70d-db9bfe35c4c2)

multiplayer room:
![image](https://github.com/ppy/osu/assets/32331609/59a1a6fb-f24f-4d17-9ddf-52a0a8328911)

**Comment by peppy** [comment, marked as answer]
I agree with changing "freemods" as it is not a word. But the term "free mods" doesn't make sense to a player as much as the host.

Maybe "optional mods" works better in both places.
